module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -190,8 +193,8 @@ namespace behaviour {
                         bool objectsMissing = false;
 
                         // Get the list of objects which are currently visible
-                        std::vector<Ball> ballFixationObjects = getFixationObjects(vballs, objectsMissing);
-                        std::vector<Goal> goalFixationObjects = getFixationObjects(vgoals, objectsMissing);
+                        Balls ballFixationObjects = getFixationObjects(vballs, objectsMissing);
+                        Goals goalFixationObjects = getFixationObjects(vgoals, objectsMissing);
```

Comment:
I'm pretty sure these two can stay as a vector of Balls/Goals as these are not sent out as a message.